Formula/csound.rb

```diff
@@ -71,19 +71,17 @@ def install
 
     resource("getfem").stage { cp_r "src/gmm", buildpath }
 
-    args = std_cmake_args + %W[
-      -DBUILD_JAVA_INTERFACE=ON
-      -DBUILD_LINEAR_ALGEBRA_OPCODES=ON
-      -DBUILD_LUA_INTERFACE=OFF
-      -DBUILD_WEBSOCKET_OPCODE=OFF
-      -DCMAKE_INSTALL_RPATH=@loader_path/../Frameworks;#{rpath}
-      -DCS_FRAMEWORK_DEST=#{frameworks}
-      -DGMM_INCLUDE_DIR=#{buildpath}/gmm
-      -DJAVA_MODULE_INSTALL_DIR=#{libexec}
-    ]
-
     mkdir "build" do
-      system "cmake", "..", *args
+      system "cmake", "..",
+                      "-DBUILD_JAVA_INTERFACE=ON",
+                      "-DBUILD_LINEAR_ALGEBRA_OPCODES=ON",
+                      "-DBUILD_LUA_INTERFACE=OFF",
+                      "-DBUILD_WEBSOCKET_OPCODE=OFF",
+                      "-DCMAKE_INSTALL_RPATH=@loader_path/../Frameworks;#{rpath}",
+                      "-DCS_FRAMEWORK_DEST=#{frameworks}",
+                      "-DGMM_INCLUDE_DIR=#{buildpath}/gmm",
+                      "-DJAVA_MODULE_INSTALL_DIR=#{libexec}",
+                      *std_cmake_args
       system "make", "install"
     end
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build",
                    "-DBUILD_JAVA_INTERFACE=ON",
                    "-DBUILD_LINEAR_ALGEBRA_OPCODES=ON",
                    "-DBUILD_LUA_INTERFACE=OFF",
                    "-DBUILD_WEBSOCKET_OPCODE=OFF",
                    "-DCMAKE_INSTALL_RPATH=@loader_path/../Frameworks;#{rpath}",
                    "-DCS_FRAMEWORK_DEST=#{frameworks}",
                    "-DGMM_INCLUDE_DIR=#{buildpath}/gmm",
                    "-DJAVA_MODULE_INSTALL_DIR=#{libexec}",
                    *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--build", "install"
```

Reply:
Using

```ruby
system "cmake", "--build", "install"
```

resulted in

```
Error: <Homebrew buildpath>/install is not a directory
```

for me. I changed this to

```ruby
system "cmake", "--install", "build"
```

and I made this change in both places CMake is used.